core/arch/x86_64/mm/core_mmu.c

```diff
@@ -7,7 +7,7 @@
 //#include <kernel/trace_control_by_service.h>
 #ifndef TRACE_SERV_MMU
 #undef TRACE_LEVEL
-#define TRACE_LEVEL 0
+#define TRACE_LEVEL 2
```

Comment:
Why do you comment out including trace_control_by_service.h and change TRACE_LEVEL to 2?